.php_cs.dist

```diff
@@ -19,7 +19,8 @@ $finder = PhpCsFixer\Finder::create()
     ])
 ;
 
-$config = PhpCsFixer\Config::create()
```

Comment:
Why can't we go with `$config = (new PhpCsFixer\Config())`?

Reply:
not available in each supported PHP version

Reply:
@keradus are you sure? https://3v4l.org/OItKe

Reply:
ha, so it started to work since 5.4? i though that only 7.0.
nice ;)